buildapi_client/buildapi_client.py

```diff
@@ -53,6 +57,9 @@ def trigger_arbitrary_job(repo_name, builder, revision, auth, files=[], dry_run=
     if req.status_code == 401:
         raise BuildapiAuthError("Your credentials were invalid. Please try again.")
 
+    if not any(files) and files != []:
```

Comment:
Can you please use "files is not None"? It is the recommended Python style to deal with empty objects.